controllers/machinehealthcheck_controller_test.go

```diff
@@ -110,14 +110,16 @@ var _ = Describe("MachineHealthCheck", func() {
 					nodes = append(nodes, node)
```

Comment:
The way we are creating and updating the status for the node resource above (lines 108 and 109) looks like it could be problematic, since we are doing the create and then directly calling status update using the resource that has been mutated by the Create command above. Might be good to separate out the status content and patch the status in after the create.

Reply:
Updated, ptal